src/style.css

```diff
@@ -1,3 +1,88 @@
+* {
+  /* Скидання параметрів браузера */
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  display: flex;
+}
+
+
+
+
+.logo-1200 {
+  width: 40px; /* Фіксована ширина логотипа */
+  height: auto; /* Дозволяє автоматично підлаштувати висоту */
+  display: block;
+}
+
+.imgLogo img {
```

Comment:
The class '.imgLogo' should be named to represent the content or function of the images it styles, rather than the tag and style applied.